source/elements/atoms/Grid/GridIE.css

```diff
@@ -0,0 +1,47 @@
+.Grid {
```

Comment:
Could use some documentation here explaining what this is and when it is used (because looking at this at the moment, it looks like both 'Grid.css' and 'GridIE.css' are being bundled and applied. It would be nice if we could have some Javascript look at the browser being used, and apply the correct styles when it needs to? Cleaner solution but a bit more work. Open for discussion.